src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/RegressionTest.java

```diff
@@ -238,6 +238,30 @@ public void testXmlQuery() throws SQLException {
         }
     }
 
+    /**
+     * Tests Json query
+     * 
+     * @throws SQLException
+     */
+    @Test
+    public void testJsonQuery() throws SQLException {
+        try (Connection connection = getConnection(); Statement stmt = connection.createStatement()) {
+            tableName = RandomUtil.getIdentifier("try_SQLJSON_Table");
+            TestUtils.dropTableIfExists(AbstractSQLGenerator.escapeIdentifier(tableName), stmt);
```

Comment:
We can consider enhancing this test to insert a couple of records, update the first record with a where predicate on JSON field itself, delete the second record (with a predicate), read the records back and validate that its 1 record with expected value of the json column.